packages/foam-cli/src/commands/migrate.ts

```diff
@@ -0,0 +1,97 @@
+import { Command, flags } from '@oclif/command';
+import * as ora from 'ora';
+import { initializeNoteGraph, generateLinkReferences, generateHeading, getKebabCaseFileName } from 'foam-core';
+import { applyTextEdit } from '../utils/apply-text-edit';
+import { writeFileToDisk } from '../utils/write-file-to-disk';
+import { renameFile } from '../utils/rename-file';
+import * as fs from 'fs'
+
+
+export default class Migrate extends Command {
```

Comment:
Bit of repetition here between `janitor` and `migrate`. Ideally I'd like to have these refactored a bit, but since we will soon change how the reference link definition generation works, we can do that as a second pass then.

Let's leave a `@todo` here at least.

Reply:
I agree. I did some refactoring but still, there is a lot that can be done there. I have added a @todo for now.